controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -814,7 +814,21 @@ void cli_bml::events_update_to_socket_cb(const struct BML_EVENT *event)
     cli_bml::events_update_cb(event, false);
 }
 
-void cli_bml::wfa_ca_reply_to_console_cb(const char *buffer) { std::cout << buffer << std::endl; }
+void cli_bml::wfa_ca_reply_to_console_cb(BML_CTX ctx, const char *buffer)
+{
+    cli_bml *pThis = reinterpret_cast<cli_bml *>(bml_get_user_data(ctx));
+    if (!pThis) {
+        std::cout << "ERROR: Internal error - invalid context!" << std::endl;
+        return;
+    }
+
+    static const std::string ref("status,RUNNING");
+    std::string reply(buffer);
+    pThis->pending_response =
+        ((reply.size() >= ref.size()) && (reply.substr(0, ref.size()) == ref)) ? true : false;
```

Comment:
Or maybe it is because you review it late at night 😉 
Even if I'll pass it in the callback, this piece of code will just move to somewhere else.
At first, I passed it on the BML WFA-CA response message but you guys asked me to remove it.
Basically what I did here is to check if the response message stat with "status,RUNNING"
Maby I will add a comment that explains what I did here, what do think?

Reply:
Added a comment